Concentration/Concentration.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Concentration.swift
+//  Concentration
+//
+//  Created by Oren Dinur on 08/02/2022.
+//
+
+import Foundation
+
+class Concentration{
+    
+    var cards = Array <Card>()
+    //Same as [Card]()
+    
+    var flipCount = 0
+    
+    var score = 0
+    
+    var seenCards = [Int:Int]()
+
+    var indexOfOneAndOnlyFaceUpCard: Int?
+     
+    //The beginning of our game
+    var previousClick = Date()
+
+    func chooseCard(at index: Int){
```

Comment:
This method is very long. Try to extract some of the logic to some separate methods. It will make the code much more readable.